src/main/java/mate/academy/spring/model/dto/response/UserResponseDto.java

```diff
@@ -0,0 +1,31 @@
+package mate.academy.spring.model.dto.response;
+
+public class UserResponseDto {
+    private Long id;
+    private String email;
+    private String password;
```

Comment:
**Don’t expose secure data in your ResponseDtos.**
https://mate-academy.github.io/jv-program-common-mistakes/java-spring/rest-practice/jv-spring-rest-practice_checklist